CONTRIBUTING.md

```diff
@@ -1,107 +1,121 @@
+   lessons or workshops stored in `_includes` or `_layouts`, please work in
+   <https://github.com/carpentries/styles>.
 
 ## What to Contribute
 
-There are many ways to contribute, from writing new exercises and improving existing ones to
-updating or filling in the documentation and submitting [bug reports][issues] about things that
-don't work, aren't clear, or are missing. If you are looking for ideas, please see the 'Issues' tab
-for a list of issues associated with this repository, or you may also look at the issues for
-[Data Carpentry][dc-issues] and [Software Carpentry][swc-issues] projects.
+There are many ways to contribute, from writing new exercises and improving
+existing ones to updating or filling in the documentation and submitting [bug
+reports][issues] about things that don't work, aren't clear, or are missing. If
+you are looking for ideas, please see the 'Issues' tab for a list of issues
+associated with this repository, or you may also look at the issues for [Data
+Carpentry][dc-issues] and [Software Carpentry][swc-issues] projects.
 
-Comments on issues and reviews of pull requests are just as welcome: we are smarter together than we
-are on our own. Reviews from novices and newcomers are particularly valuable: it's easy for people
-who have been using these lessons for a while to forget how impenetrable some of this material can
-be, so fresh eyes are always welcome.
+Comments on issues and reviews of pull requests are just as welcome: we are
+smarter together than we are on our own. Reviews from novices and newcomers are
+particularly valuable: it's easy for people who have been using these lessons
+for a while to forget how impenetrable some of this material can be, so fresh
+eyes are always welcome.
 
 ## What *Not* to Contribute
 
-Our lessons already contain more material than we can cover in a typical workshop, so we are usually
-*not* looking for more concepts or tools to add to them. As a rule, if you want to introduce a new
-idea, you must (a) estimate how long it will take to teach and (b) explain what you would take out
-to make room for it. The first encourages contributors to be honest about requirements; the second,
-to think hard about priorities.
+Our lessons already contain more material than we can cover in a typical
+workshop, so we are usually *not* looking for more concepts or tools to add to
+them. As a rule, if you want to introduce a new idea, you must (a) estimate how
+long it will take to teach and (b) explain what you would take out to make room
+for it. The first encourages contributors to be honest about requirements; the
+second, to think hard about priorities.
 
-We are also not looking for exercises or other material that only run on one platform. Our workshops
-typically contain a mixture of Windows, macOS, and Linux users; in order to be usable, our lessons
-must run equally well on all three.
+We are also not looking for exercises or other material that only run on one
```

Comment:
Not sure this paragraph is as applicable to HPC Carpentry as it would be to SC, DC, or LC. Most of what we talk about is on a Linux HPC, with the exception of the Pi-calculating Python code that starts out on the local computer.

Reply:
For now, CONTRIBUTING is boilerplate taken from The Carpentries. It would be worth editing for a better fit, but that discussion belongs in a separate issue.